che-editors.yaml

```diff
@@ -234,12 +280,19 @@ editors:
       - name: intellij-novnc
         image: "quay.io/che-incubator/che-editor-intellij-community:2020.2-cdc002c"
         mountSources: true
-        volumes:
+        volumeMounts:
           - name: idea-configuration
             path: "/JetBrains/IdeaIC"
+        ports:
+         - exposedPort: 8080
         memoryLimit: "2048M"
-  - id: ws-skeleton/che-editor-eclipseide/4.9.0
-    endpoints:
+  - id: ws-skeleton/eclipseide/4.9.0
+    displayName: Eclipse IDE
+    description: Eclipse running on the Web with Broadway
+    title: Eclipse IDE (in browser using Broadway) as editor for Eclipse Che
+    icon: https://cdn.freebiesupply.com/logos/large/2x/eclipse-11-logo-svg-vector.svg
+    repository: https://github.com/ws-skeleton/che-editor-eclipseide/
+    firstPublicationDate: "2019-02-05"    endpoints:
```

Comment:
should be formatted. `endpoints` is a name of the attribute

Reply:
good catch 👍 , fixed